plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -4,6 +4,12 @@ const wcApi = require( '@woocommerce/woocommerce-rest-api' ).default;
 const virtualProductName = 'Virtual Product Name';
 const nonVirtualProductName = 'Non Virtual Product Name';
 const productPrice = '9.99';
+const salePrice = '6.99';
+const productDescription = 'Virtual product description.';
+const productSKU = '1234567890'
+const productPurchaseNote = 'Virtual product purchase note'
+const productAttribute = 'color'
+const productAttributeColor = 'red | white'
```

Comment:
I would suggest adding assertions for these values later on in the test.  So far, the only assertion we have on this test is that the "Product published" notification was displayed. Adding more assertions around these newly added details will make the test more, well... assertive 😅

Reply:
Good idea @rodelgc , if I'm not mistaken, this aligns with my [comment - second point -](https://github.com/woocommerce/woocommerce-quality/issues/607#issuecomment-1898576008) in the related issue.

tl;dr: My impression is that this script automates the creation of a product, but there's no test afterwards to verify that the product was created with the expected values. I hope I'm explaining myself right, not 100% if this is what you meant here too.

Would you please mind sharing with me one example of assertion that can be used to verify any of those values?